### DISCUSSION TITLE
ThinQ Oven Support

### DISCUSSION THREAD

**Discussion by rdnewt** [nVuln/homebridge-lg-thinq#28] [Ideas]
Are there any plans to add support for the recent LG ThinQ Ovens at some point in the future? I'm wondering specifically about cooktop and oven statuses and oven pre-heat notifications. Thanks!

**Comment by nVuln** [comment]
I don't have Ovens, but I can add it with your help, please install this plugin and turn on homebridge debug mode, you can see your device data in debug logging, then upload it here
if your device is using thing2 platform, I can implement it

**Reply by rdnewt** [reply]
It looks like it does use thinq2, so that would be great if you could add it. I've attached the logs from the ThinQ package at startup. Thanks, and please let me know if you need anything else!

[homebridge.log.txt](https://github.com/nVuln/homebridge-lg-thinq/files/6901966/homebridge.log.txt)

**Reply by nVuln** [reply]
can you capture some screenshot of Ovens in LG app?

**Reply by rdnewt** [reply]
Sure thing, here’s screenshots of the oven and oven settings in the LG app. Uploading directly from mobile so hopefully this works

![867C8D49-7497-44DF-9ED9-E5008A5D963B](https://user-images.githubusercontent.com/57018178/127739298-199b6494-7aee-4337-b9a7-b952f46c4392.jpeg)
![B5C49CF3-332E-4963-9B71-F1CDEF4E1CC1](https://user-images.githubusercontent.com/57018178/127739299-bf9eee0e-bf91-4d0d-ad96-44b095ac8043.jpeg)
![8A61603C-2277-493C-91B6-F918CBB71371](https://user-images.githubusercontent.com/57018178/127739301-0820ecd6-7067-4b82-b06a-7565cd6e5b77.jpeg)
![5270B7B8-E3CE-4E78-9D3D-E0274F4C1A5A](https://user-images.githubusercontent.com/57018178/127739302-73207e88-d2e9-40f7-88d9-dd312158c4b0.jpeg)

**Reply by nVuln** [reply]
please get device data while your ovens is running

**Reply by rdnewt** [reply]
I have attached information with the oven turned on to 350 degF, no burners on. Do you also need data with a burner on? 
[oven_on_homberidge.log.txt](https://github.com/nVuln/homebridge-lg-thinq/files/6951274/oven_on_homberidge.log.txt)

**Reply by Josh-Duncan** [reply]
I am also able to assist with the testing of the oven/range plugin if required.  I think this could be a very useful function, even if there is only status information for the appliance.

**Reply by nVuln** [reply]
please following #87 , I need collect some device data before working on it, hope you can help
